site/_data/i18n/tags.yml

```diff
@@ -23,6 +23,34 @@ autoplay:
 capabilities:
   en: Capabilities
 
+# DevTools-related tags
+css:
+  en: CSS
+
+javascript:
+  en: JavaScript
+
+get-started:
+  en: Get started
+
+prototype-fixes:
+  en: Prototype fixes
```

Comment:
this is not a common term we use as web developer

Reply:
It was suggested by the PM. I want to see how this tags does in terms of usage statistics.